guides/common/modules/proc_disabling-registration-to-insights.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Disabling Registration to Red Hat Insights
 
-{Project} is registered to Red Hat Insights by default. After {Project} installation, you can choose to unregister or register Red Hat Insights as needed. For example, if you need to use {Project} in a disconnected environment, you can unregister `insights-client` from {ProjectServer}. 
+{Project} is registered to Red Hat Insights by default. After {Project} installation or upgradation, you can choose to unregister or register Red Hat Insights as needed. For example, if you need to use {Project} in a disconnected environment, you can unregister `insights-client` from {ProjectServer}. 
```

Comment:
Just saw this now, upgradation isn't a word

Change to "After you install or upgrade {Project}"